src/jsMain/kotlin/io/beatmaps/maps/mapinfo.kt

```diff
@@ -171,7 +173,7 @@ class MapInfo : RComponent<MapInfoProps, MapInfoState>() {
                                     }
                                 }
 
-                                if (userData?.curator == true) {
+                                if (userData?.curator == true && !props.isOwner) {
```

Comment:
Probably want to add server-side validation for that

Reply:
Good idea.